server/utils/errorHandler.js

```diff
@@ -0,0 +1,17 @@
+const errorHandler = (err,req,res,next)=>{
+    const status = res.statusCode === 200 ? 500 : res.statusCode;
```

Comment:
I don't understand what this is doing. If the status code is 200 we want to return a server error?

Reply:
Thank you @ThinkTink for your time and comments. I will work on them. For this one, I actually saw when I was testing the routes with postman that sometimes server errors happen with 200 status code. I have tested this function, it won't mess up the actual 200 with no errors, this will only captures errors that have wrongly 200 status code.

Reply:
Ideally we should track down where in the code is producing a 200 when there's an error and fix it there instead